indico/modules/rb_new/client/js/modules/bookRoom/BookRoom.jsx

```diff
@@ -136,8 +146,24 @@ class BookRoom extends React.Component {
         );
     };
 
-    renderFilters(refName) {
-        const {[refName]: ref} = this.state;
+    renderFilters(refName, isTimelineVisible) {
```

Comment:
Isn't `isTimelineVisible` a prop? If so, is there a need to pass it as a parameter here?